ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -343,30 +343,55 @@ private void drawRectangularBackgroundWithBorders(Canvas canvas) {
       int width = getBounds().width();
       int height = getBounds().height();
 
+      // if the path drawn previously is of the same color,
+      // there would be a slight white space between each border
+      // after drawing is done, we will re-enable anti-alias
+      mPaint.setAntiAlias(false);
+
       if (borderLeft > 0 && colorLeft != Color.TRANSPARENT) {
         mPaint.setColor(colorLeft);
-        canvas.drawRect(0, borderTop, borderLeft, height - borderBottom, mPaint);
+        Path leftBorderPath = new Path();
```

Comment:
Instead of allocating a new Path() each time, can you reset() and reuse?  Generally better to avoid allocations while drawing.